source/render/ODFrameListener.cpp

```diff
@@ -92,8 +92,6 @@ ODFrameListener::ODFrameListener(Ogre::RenderWindow* renderWindow, Ogre::Overlay
 
     mGameMap = new GameMap(false);
 
-    mMiniMap = new MiniMap(mGameMap);
```

Comment:
True enough. We still have a bit of history here. But at least, you untangled the minimap, framelistener and the camera manager code. :)
The next interesting thing IMHO is to reduce the minimap update/draw calls to their minimum. There might be a good perf impact.